client/state/sites/terms/actions.js

```diff
@@ -0,0 +1,66 @@
+/**
```

Comment:
I think we'll want `terms` to be a top-level folder in `state`. Trying to think of the rationale, I don't recall why this choice was made in the first place, but it may have been influenced by a desire to track items as global entities (e.g. post by global ID). Also, nesting is a bit more difficult to navigate when trying to find a specific subtree. That said, it can be useful in some cases to provide context (e.g. the future `state.postTypes.taxonomies` subtree, since taxonomies have different meaning specific to post types than at a top-level).